bin/import-wa-bpmn-diagram.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eu
+workspace=${1}
+
+serviceToken=$($(realpath $workspace)/bin/utils/idam-lease-service-token.sh civil_service \
+  $(docker run --rm toolbelt/oathtool --totp -b ${S2S_SECRET:-AABBCCDDEEFFGGHH}))
+
+filepath="$(realpath $workspace)/resources"
+
+for file in $(find ${filepath} -name '*.bpmn')
+do
+  uploadResponse=$(curl --insecure -v --silent -w "\n%{http_code}" --show-error -X POST \
+    ${CAMUNDA_BASE_URL:-http://localhost:9404}/engine-rest/deployment/create \
+    -H "Accept: application/json" \
+    -H "ServiceAuthorization: Bearer ${serviceToken}" \
+    -F "deployment-name=$(date +"%Y%m%d-%H%M%S")-$(basename ${file})" \
+    -F "file=@${filepath}/$(basename ${file})")
+
+upload_http_code=$(echo "$uploadResponse" | tail -n1)
+upload_response_content=$(echo "$uploadResponse" | sed '$d')
+
+if [[ "${upload_http_code}" == '200' ]]; then
+  echo "$(basename ${file}) diagram uploaded successfully (${upload_response_content})"
+  continue;
+fi
+
+echo "$(basename ${file}) upload failed with http code ${upload_http_code} and response (${upload_response_content})"
+continue;
```

Comment:
Same as above. Seems risky to `continue;` here. Would be safer to `exit 1;` instead and fail the build.

Reply:
This one is ok as there is only 1 bpmn from wa side

Reply:
So it's ok if the bpmn is not loaded correctly and therefore whatever functionality it's supposed to drive fails even though the build succeeded?